src/components/Profile/Profile.tsx

```diff
@@ -1,6 +1,15 @@
-import * as S from "./Profile.style";
+import * as S from './Profile.style';
 
-function Profile({ data }) {
+interface Props {
+  data: {
```

Comment:
해당 내부 프로퍼티를 모두 optional 처리해줄수밖에없는 이유가 있을까요?
```suggestion
  data?: {
```
이런식으로 data가 optional props라는걸 명시해주면 내부 필드를 전부 optional 처리 하지 않으셔도됩니다!